invitations/models.py

```diff
@@ -17,27 +17,46 @@
 
 
 @python_2_unicode_compatible
-class Invitation(models.Model):
-
-    email = models.EmailField(unique=True, verbose_name=_('e-mail address'),
-                              max_length=app_settings.EMAIL_MAX_LENGTH)
+class BaseInvitation(models.Model):
```

Comment:
Thanks for review and suggestions :+1: 
( Anyway this is just very WIP :), so flake8, right placement etc. are coming :) )